docs/radio.rst

```diff
@@ -132,6 +133,27 @@ Functions
 
     A ``ValueError`` exception is raised if conversion to string fails.
 
+.. py:function:: receive_all()
+
+    Returns a tuple containing four values representing the next incoming
+    message on the message queue. If there no pending messages then ``None``
+    is returned.
+
+    The four values in the tuple represent:
+
+    * the next incoming message on the message queue as bytes.
+    * the RSSI signal strength: a value between 0 (strongest) and 255 (weakest).
+    * the approximate distance: equivalent to ``math.pow(10, (rssi - tx_power_at_1m) / (-10 * path_loss_exponent))``
```

Comment:
I would suggest taking this formula out. The reason being that the calculation of distance with RSSI is done by calculating the difference between the power at transmit and the power at receive. Right now the library does not send the details of what power level the data was transmitted at.